core/block.js

```diff
@@ -890,63 +874,10 @@ Blockly.Block.prototype.setTooltip = function(newTip) {
  * @return {string} #RRGGBB string.
  */
 Blockly.Block.prototype.getColour = function() {
+  // TODO: Can we remove this?
```

Comment:
Open question: can I remove colour and hue from block entirely?

Reply:
It doesn't need to be stored internally on the block, but a Field could need to call into the block to ask it for it's colour. So I think we still need references to all three of these.

Reply:
I had those fields looking at the block's colourer--now they look at the path object.

Reply:
Yeah, I'd rather not surface that path object outside of the renderer. It makes it public domain if fields are accessing it.

Reply:
Moving back to field getting colours from the block.

Reply:
And I've filed #3360 about removing colour from block and having it only on blockSvg.